Instagram/UserProfile/Controllers/EditProfile/EditProfileViewController.swift

```diff
@@ -20,23 +25,207 @@ class EditProfileViewController: UIViewController, UITableViewDelegate, UITableV
+            return cell
+        case 3:
+            let cell = tableView.dequeueReusableCell(withIdentifier: EditWebsiteTableViewCell.identifier, for: indexPath) as! EditWebsiteTableViewCell
+            cell.firstLabel.text = "Website"
+            return cell
+        case 4:
+            let cell = tableView.dequeueReusableCell(withIdentifier: EditProfileTableViewCell.identifier, for: indexPath) as! EditProfileTableViewCell
+            cell.firstLabel.text = "Bio"
+//            cell.secondLabel.isUserInteractionEnabled = true
+//            let tap = UITapGestureRecognizer(target: self, action: #selector(chanegeBioMethod(tapGestureRecognizer:)))
+//            cell.secondLabel.addGestureRecognizer(tap)
+            return cell
+        case 5:
+            let cell = tableView.dequeueReusableCell(withIdentifier: EditProfileOtherTableViewCell.identifier, for: indexPath) as! EditProfileOtherTableViewCell
+            cell.firstLabel.text = "Switch to professional Account"
+            cell.firstLabel.textColor = .tintColor
+            return cell
+        case 6:
+            let cell = tableView.dequeueReusableCell(withIdentifier: EditProfileOtherTableViewCell.identifier, for: indexPath) as! EditProfileOtherTableViewCell
+            cell.firstLabel.text = "Create avatar"
+            cell.firstLabel.textColor = .tintColor
+            return cell
+        case 7:
+            let cell = tableView.dequeueReusableCell(withIdentifier: EditProfileOtherTableViewCell.identifier, for: indexPath) as! EditProfileOtherTableViewCell
+            cell.firstLabel.text = "Personal information settings"
+            cell.firstLabel.textColor = .tintColor
+            return cell
+        default:
+            return UITableViewCell()
+        }
+        
+    }
+    
+    func tableView(_ tableView: UITableView, didSelectRowAt indexPath: IndexPath) {
+        switch indexPath.row
+        {
+        case 0:
+            let vc = EditProfileNameViewController()
+            navigationController?.pushViewController(vc, animated: true)
+        case 1:
+            let vc = EditProfileUsernameViewController()
+            navigationController?.pushViewController(vc, animated: true)
+        case 2:
+            let vc = EditProfilePronounsViewController()
+            navigationController?.pushViewController(vc, animated: true)
+        case 4:
+            let vc = EditProfileBioViewController()
+            navigationController?.pushViewController(vc, animated: true)
+        case 5:
+            let vc = ProfessionalAccountViewController()
+            navigationController?.pushViewController(vc, animated: true)
+        case 6:
+            let vc = AvatarViewController()
+            navigationController?.pushViewController(vc, animated: true)
+        case 7:
+            let vc = personalInformationViewController()
+            navigationController?.pushViewController(vc, animated: true)
+        default:
+            print("do nothing")
```

Comment:
use print on DEBUG only
```suggestion
            print("do nothing")
```

Reply:
#if DEBUG
    println("do nothing")
#else
    println("I'm running in a non-DEBUG mode")
#endif